packages/component/src/Styles/StyleSet/TextContent.js

```diff
@@ -24,6 +24,15 @@ export default function createTextContentStyle({ bubbleMaxWidth, bubbleMinHeight
       '& pre': {
         overflow: 'hidden'
       }
+    },
+    '& .externalLink': {
+      margin: 0,
+      padding: 0,
+      backgroundPosition: 'center right',
+      backgroundRepeat: 'no-repeat',
+      backgroundImage:
+        'linear-gradient(transparent,transparent),url(https://en.wikipedia.org/w/skins/Vector/resources/skins.vector.styles/images/external-link-ltr-icon.svg?b4b84)',
```

Comment:
There is likely a media process that must be followed. Legally the image must be able to be used in open source. Images should not be linked to other sources as this causes a many issues.

It look like the image should follow this [similar pattern](https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/component/src/Styles/StyleSet/TypingAnimation.js) and should also be embedded using base64 encoding.

Reply:
Please check and use the Fabric UI, which is the source of all of our current icons.

Reply:
Hi @corinagum, can you provide me with an example on how to properly use the Fabric UI? I can't find something similar on the webchat code, other than the example Eric is providing with the base64 images

Reply:
For example, when I see how the Microphone or Attachment icons are being used, I see that they are SVGs:
```html
<svg height="28" viewBox="0 0 25.75 46" width="28"><path clip-rule="evenodd" d="M20.75 11.75v21.37a7.69 7.69 0 0 1-.62 3.07 7.95 7.95 0 0 1-4.19 4.19 7.89 7.89 0 0 1-6.13 0 7.95 7.95 0 0 1-4.19-4.19 7.69 7.69 0 0 1-.62-3.07v-22.5a5.27 5.27 0 0 1 .45-2.17 5.69 5.69 0 0 1 3-3 5.48 5.48 0 0 1 4.35 0 5.69 5.69 0 0 1 3 3 5.27 5.27 0 0 1 .45 2.17v22.5a3.41 3.41 0 0 1-.26 1.32 3.31 3.31 0 0 1-1.8 1.8 3.46 3.46 0 0 1-2.63 0 3.31 3.31 0 0 1-1.8-1.8 3.41 3.41 0 0 1-.26-1.32V14h2.25v19.12a1.13 1.13 0 1 0 2.25 0v-22.5a3.4 3.4 0 0 0-.26-1.31 3.31 3.31 0 0 0-1.8-1.8 3.46 3.46 0 0 0-2.63 0 3.31 3.31 0 0 0-1.8 1.8 3.4 3.4 0 0 0-.26 1.31v22.5a5.32 5.32 0 0 0 .45 2.18 5.69 5.69 0 0 0 3 3 5.48 5.48 0 0 0 4.35 0 5.69 5.69 0 0 0 3-3 5.32 5.32 0 0 0 .45-2.18v-21.37z"></path></svg>
```

Reply:
Commit 4dd74342123906411fa3494df2ef6368ccbb3a8d resolves this as @compulim has provided me an approved Office UI Fabric icon in svg format.
Said svg icon is encoded in base64 and used